File: Backend/services/core/src/main/java/org/edu_sharing/restservices/node/v1/model/WorkflowHistory.java
package org.edu_sharing.restservices.node.v1.model;

import lombok.Data;
import org.edu_sharing.restservices.shared.Authority;
import org.edu_sharing.restservices.shared.UserSimple;

@Data
public class WorkflowHistory {
	private long time;
	private UserSimple editor;
	private Authority[] receiver;
	private String status;
	private String comment;
}

File: Frontend/projects/edu-sharing-api/src/lib/api/models/workflow-history.ts
/* tslint:disable */
/* eslint-disable */
import { Authority } from './authority';
import { UserSimple } from './user-simple';
export interface WorkflowHistory {
    comment?: string;
    editor?: UserSimple;
    receiver?: Array<Authority>;
    status?: string;
    time?: number;
}
